modules/navier_stokes/test/tests/finite_volume/wcns/boundary_conditions/flux_bcs_velocity.i

```diff
@@ -258,7 +258,7 @@ inlet_velocity = 0.001
     type = Receiver
     default = ${inlet_velocity}
   []
-  [surface_inlet]
+  [area_pp_left]
```

Comment:
Why this change (and the one in the action input)? I assume these were the reasons for the re-gold?

Reply:
This is actually a bug fix, in certain scenarios we didn't create area PPs in the action automatically for the flux BCs (scalar transport needs an area PP with the mass flux too to infer velocity). Now we create PPs every time a flux-bc is used. However the automatic name of the PP is `area_pp_${bd_name}` and I want to compare the action and non-action results to the same exodus file so I had to regold.